src/lib/import-export/import/importers/importer.ts

```diff
@@ -83,7 +79,7 @@ export abstract class BaseImporter extends EventEmitter implements Importer {
 	}
 }
 
-export class JetpackImporter extends BaseImporter {
+export abstract class BaseBackupImporter extends BaseImporter implements Importer {
```

Comment:
It would be better to use composition over inheritance, but currently, we rely on inheritance here, so I followed that pattern and added another level of base class.